pkg/util/webhooks/tls_test.go

```diff
@@ -78,8 +78,8 @@ var _ = Describe("TLS", func() {
 	})
 
 	table.DescribeTable("on virt-handler should", func(serverSecret, clientSecret string, errStr string) {
-		serverTLSConfig := webhooks.SetupTLSForVirtHandlerServer(caManager, certmanagers[serverSecret])
-		clientTLSConfig := webhooks.SetupTLSForVirtHandlerClients(caManager, certmanagers[clientSecret])
+		serverTLSConfig := webhooks.SetupTLSForVirtHandlerServer(caManager, certmanagers[serverSecret], true)
```

Comment:
How about testing the `false` case? I don't want to sound so picky, but in case the logic gets more complicated, we might already have that branch covered?

Reply:
Sure thing! Testing the false case will make sure the logic always performs for default cases.